pkg/config/config_test.go

```diff
@@ -131,7 +132,8 @@ func Test_ExtractConfigData(t *testing.T) {
  seedNode: true
  segreaderNode: true
  metareaderNode: true
- segFlushIntervalSecs: 1200
+ idleWipFlushIntervalSecs: 1200
+ maxWaitWipFlushIntervalSecs: 300
```

Comment:
same here

Reply:
yes , a error log will be printed and we get set to default. Since this is just for testing , then we should set correct value (just a nit btw)

Reply:
yes we are testing this particular scenario where we will give more than allowed values and expect to see the correct allowed values.